### PR TITLE
test: added test for get pod's logs

### PR DESCRIPTION
**Pull request by matejvasek**
# Changes

- :test_tube: Test get pod's logs.

/kind test

### CODE REVIEW

**Comment by matejvasek** [comment]
@zroubalik @dolfolife

File: k8s/logs_test.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	}
+
+	pod, err = cliSet.CoreV1().Pods(testingNS).Create(ctx, pod, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Log("created pod: " + testingPodName)
+
+out:
+	for i := 0; i < 600; i++ {
```
Comment (lance) [inline k8s/logs_test.go:71]: Is there not an kube API call to wait until the pod is ready?

Comment (matejvasek) [reply k8s/logs_test.go:71]: It is quite low level. I am aware only of watch interface which is not much better than this with respect to line of code.

Comment (matejvasek) [reply k8s/logs_test.go:71]: https://github.com/knative/func/blob/84741e24794e039e17dd2d8292bf37502f376c88/k8s/persistent_volumes.go#L168-L193
Is it any better?

Comment (lance) [reply k8s/logs_test.go:71]: > Is it any better?

I would not be surprised if it's doing something very similar.

Comment (matejvasek) [reply k8s/logs_test.go:71]: >I would not be surprised if it's doing something very similar.

Nah I think it is smarter than that, like doing some long polling on server or something.

**Pull request merged**